Create skv wagon (#17)
hitobito/hitobito_skv: README.md

> # hitobito_skv
> A hitobito wagon defining the organization hierarchy and additional features for Swiss Canoe
> Basis Setup 
> 
> # Hitobito Skv
> 
> This hitobito wagon defines the organization hierarchy with groups and roles
> of Skv.

Reviewer: ```suggestion
# hitobito skv

A hitobito wagon defining the organization hierarchy and additional features for Swiss Canoe.
(Schweizerischer Kanu-Verband)
```
